test/helper.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
```

Comment:
This file will be run by the wildcard in package.json. Can you rename `test/basic.js` into `test/basic.test.js` and adjust that pattern? Thanks